core/src/main/kotlin/com/exactpro/th2/read/db/core/ValueTransformerFactory.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2024 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactpro.th2.read.db.core
+
+interface ValueTransformerFactory: ValueTransformer {
+    /**
+     * Holds database type of JDBC URI protocol
+     */
+    val dbType: String
```

Comment:
note: I would say that this it rather db vendor than db type. But have no objections if you keep this naming